sdk/src/main/scala/com/horizen/account/state/AccountStateView.scala

```diff
@@ -192,17 +195,38 @@ class AccountStateView(private val metadataStorageView: AccountStateMetadataStor
     val processor = messageProcessors.find(_.canProcess(message, this)).getOrElse(
       throw new IllegalArgumentException(s"Transaction ${ethTx.id} has no known processor.")
     )
+
+    val receipt = new EthereumReceipt()
+    receipt.setTransactionType(ethTx.version())
+    receipt.setTransactionHash(txHash)
+
+    if (!isEoaAccount(ethTx.getTo.address()) )
```

Comment:
Possible Nullpointer exception if getTo is null

Reply:
Fixed, thanks